docs/src/App.js

```diff
@@ -17,7 +17,7 @@ const App = ({ componentMenu, versions }) => (
            * We can't place <Sidebar /> inside of <Routes /> because it will be remounted on page
            * switch. We also don't want to show <Sidebar /> for layouts pages and maximized pages.
            */}
-          <Route path='/layouts/*' component={null} />
+          <Route exact path='/layouts/:name' component={null} />
```

Comment:
Because `/layouts` because same the same prefix, we had match there and sidebar was hidden 💥